Sources/Player/OfflineEngine/Internal/Storage/OfflineEntry.swift

```diff
@@ -0,0 +1,104 @@
+import AVFoundation
+import Foundation
+
+struct OfflineEntry: Codable {
+	let productId: String
+	let productType: ProductType
+	let assetPresentation: AssetPresentation
+	let audioMode: AudioMode?
+	let audioQuality: AudioQuality?
+	let audioCodec: AudioCodec?
+	let audioSampleRate: Int?
+	let audioBitDepth: Int?
+	let videoQuality: VideoQuality?
+	var revalidateAt: UInt64?
+	var expiry: UInt64?
+	let mediaType: String?
+	let albumReplayGain: Float?
+	let albumPeakAmplitude: Float?
+	let trackReplayGain: Float?
+	let trackPeakAmplitude: Float?
+	var mediaUrl: URL?
+	var licenseUrl: URL?
```

Comment:
[suggestion] minor, just noticed `Url` instead of `URL`, since it's an acronym:
```suggestion
	var mediaURL: URL?
	var licenseURL: URL?
```
If we rename it, there are more occurrences but I've just commented here.